### ISSUE TITLE
Popup playback takes a significant amount of time to load the video.

### ISSUE THREAD

**Issue by netizeni** [TeamNewPipe/NewPipe#8160]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.22.1

### Steps to reproduce the bug

Play any video in popup mode.

### Expected behavior

Video starts playing after a couple seconds.

### Actual behavior

Video starts playing after 25+ seconds.

### Affected Android/Custom ROM version

Android 9

### Affected device model

Samsung S8

### Additional information

I noticed this before the current latest version (I'm not sure when exactly), but I always thought it's due to my internet. Today I saw that playing any video by clicking on it or playing any video in the background takes only a couple seconds to load and start playing, meanwhile when I pick to play in the popup, it always takes 25+ seconds after which it starts playing.

**Comment by pc00per** [comment]
Might be the duplicate of https://github.com/TeamNewPipe/NewPipe/issues/7630
If so, it's already fixed. Just need to wait for the next update, donno when that is.

**Comment by netizeni** [comment]
I saw that issue you opened, but I'm not sure is it a duplicate. In my case, opening a video normally or in a background takes only a couple of seconds to start playing.

If no other options, guess I will wait for the next update and see if it will help.

**Comment by blackholeearth** [comment]
when watching videos Player is trying to  download 
In this order ;

1) comments, 
2) related videos
3) video title description video cover image
***Wait all of them to load ***
4) Then video starts to download. And play.

Solution is:
1)  video starts to download. And play.
      *** Wait until video buffer at least 30sec or 1min. User can watch the start.
2) comments, 
3) related videos
4) video title description video cover image


This is present in v23   too.  The problem cant be fixed,  by reducing buffer size alone.

**Comment by opusforlife2** [comment]
> Player is trying to download
In this order ;

From what I remember, they are fetched in parallel. The smaller stuff just loads faster, for obvious reasons.
